test/e2e/toDoListFeatures.js

```diff
@@ -0,0 +1,23 @@
+describe('ToDo List', function () {
+
+  var addTaskField  = element(by.model("listCtrl.taskDescription"));
+  var addTaskButton = element(by.id('submitTask'));
+  var completeTaskCheck = element(by.model('task.completed'));
+  var tasks = element.all(by.repeater('task in listCtrl.taskList.tasks'));
+
+  beforeEach(function() {
+    browser.get('http://localhost:8080');
+  });
+
+  it('can add a task to task list', function () {
+    addTaskField.sendKeys('challenge');
```

Comment:
Maybe use a more exciting to do than "challenge". Something like 'Feed the cat'. Makes it more vivid.